### PR TITLE
C1028.0900 minor fixes

### PR DESCRIPTION
**Pull request by ClaraDopler**
This is my very limited description.

### CODE REVIEW

File: testing/test_app.py [hunk 1]
```diff
@@ -3,10 +3,7 @@
 
 from pytest_mqtt import __version__
 
-
-
-# just a random comment
-# is this better?
+# this is a simple test of the versions
```
Comment (BethanyJ123) [inline testing/test_app.py:6]: I don't think this is needed.  you should remove this line.

Comment (DelilahMichaels) [inline testing/test_app.py:6]: No you should keep it.

**Review by Ann-Beck — APPROVED** [review]

**Review by DelilahMichaels — CHANGES_REQUESTED** [review]

**Review by BethanyJ123 — APPROVED** [review]
I think it is fine

File: testing/test_app.py (hunk 1 above)
Comment (BethanyJ123) [reply testing/test_app.py:6]: I don't think it matters.  Do whatever you want.

Comment (ClaraDopler) [reply testing/test_app.py:6]: OK, I'll keep it as is than.

**Review by DelilahMichaels — APPROVED** [review]
OK it looks good now, merge away.

**Pull request merged**